backend/src/jest/helpers.js

```diff
@@ -15,3 +15,9 @@ export async function login(variables) {
     authorization: `Bearer ${response.login}`,
   }
 }
+
+//* This is a fake ES2015 template string, just to benefit of syntax
+// highlighting of `gql` template strings in certain editors.
+export function gql(strings) {
+  return strings.join('')
+}
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-07-15T19:48:51Z" title="Monday, July 15th 2019, 9:48:51 pm +02:00">Jul 15, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -15,3 +15,9 @@ export async function login(variables) {
     authorization: `Bearer ${response.login}`,
   }
 }
+
+//* This is a fake ES2015 template string, just to benefit of syntax
+// highlighting of `gql` template strings in certain editors.
+export function gql(strings) {
+  return strings.join('')
+}
```


I like this solution, but how do you feel about it @roschaefer 
don't want you to feel like you are making too many concessions
I am also ok dealing with no syntax highlighting, not sure how @Tirokk feels.